# The older libSystem in SLPPC is missing a symbol that Release SL has.

--- src/ld/Options.cpp
+++ src/ld/Options.cpp
@@ -3392,7 +3392,7 @@
 	//			Mac OS X 10.6 or later
 	//			iOS 3.1 or later
 	if ( fMakeCompressedDyldInfo ) {
-		if ( !minOS(ld::mac10_6, ld::iOS_3_1) )
+		if ( !minOS(ld::mac10_7, ld::iOS_3_1) )
 			fMakeCompressedDyldInfo = false;
 	}
 
